students/BrianNisonger/session4/trigram.py

```diff
@@ -0,0 +1,59 @@
+"""
+trigrams
+"""
+"""
+set default
+"""
+def make_words(source):
+    words=""
+    with open(source) as f:
+        lines=f.readlines()
+    punct=[",",".","-","(",")","\n","!","'s","'","?","_",";"]   
+    for line in lines:
+        for punc in punct:
+            line=line.replace(punc," ")
+        words=words+line
+    words=words.replace("  "," ")
+    words=words.replace('"', '')
+    words=words.lower()
+    return words.split()
+        
+
+
+def make_trigrams(words):
+    tris ={}
+    for w1,w2,w3 in zip(words[:-2],words[1:-1],words[2:]):
+        pair=(w1,w2)
+        if pair in tris:
+            tris[pair].append(w3)
+        else:    
+            tris[(w1, w2)]=[w3]
+    return tris
+
+def make_story(tris,new_words,story):
```

Comment:
you want the next word to be chosen based on the last 2 words in the story.

so you should only need to pass in the story here -- you can kick it off with a two word "story"